server/datastore/mysql/hosts.go

```diff
@@ -2023,6 +2023,13 @@ func (ds *Datastore) UpdateOSVersions(ctx context.Context) error {
 		statsByTeamID[0] = append(statsByTeamID[0], os)
 	}
 
+	// nothing to do so return early
+	if len(statsByTeamID) < 1 {
+		// log to help troubleshooting in case this happens
+		level.Info(ds.logger).Log("err", fmt.Sprintf("Cannot update aggregated stats for os versions: Check for records in operating_systems and host_perating_systems."))
```

Comment:
no need for `fmt.Sprintf` here. And not sure if it should be logged as an `"err"`, maybe `"msg"`?